### ISSUE TITLE
dEdX parameter updates: impact on PID

### ISSUE THREAD

**Issue by rete** [iLCSoft/ILDConfig#55]
Following the recent updates on dEdX parameters from @MasakazuKurata in the Compute_dEdXProcessor config #53 , I wonder if the parameters provided in the PIDLikelihoodProcessor : 
```xml
<parameter name="dEdxParameter_electron" type="FloatVec">
     -2.40638e-03 7.10337e-01 2.87718e-01 -1.71591e+00 0.0
</parameter>
<parameter name="dEdxParameter_muon" type="FloatVec">
      8.11408e-02 9.92207e-01 7.58509e+05 -1.70167e-01 4.63670e-04
</parameter>
<parameter name="dEdxParameter_pion" type="FloatVec">
      8.10756e-02 -1.45051e+06 -3.09843e+04 2.84056e-01 3.38131e-04
</parameter>
<parameter name="dEdxParameter_kaon" type="FloatVec">
      7.96117e-02 4.13335e+03 1.13577e+06 1.80555e-01 -3.15083e-04
</parameter>
<parameter name="dEdxParameter_proton" type="FloatVec">
      7.78772e-02 4.49300e+04 9.13778e+04 1.50088e-01 -6.64184e-04
</parameter>

<!--dE/dx normalization-->
<parameter name="dEdxNormalization" type="float"> 1.350e-7 </parameter>

<!--dE/dx error factor-->
<parameter name="dEdxErrorFactor" type="float"> 7.55 </parameter>
```
from the [HLR config](https://github.com/iLCSoft/ILDConfig/blob/master/StandardConfig/production/HighLevelReco/HighLevelReco.xml#L106) are still correct ?

**Comment by rete** [comment]
@MasakazuKurata Can you check this please ?

**Comment by MasakazuKurata** [comment]
Dear Remi
Parameters are OK for v01-19-05
But plese wait for activating smearing because I found a bug.
I will fix parameters and bug.
Thanks
Masakazu Kurata



----- Original Message -----
>From: Eté Rémi <notifications@github.com>
>To: iLCSoft/ILDConfig <ILDConfig@noreply.github.com> 
>Cc: MasakazuKurata <kurata@icepp.s.u-tokyo.ac.jp>; Mention <mention@noreply.github.com>
>Date: 2018/2/16, Fri 01:28
>Subject: [iLCSoft/ILDConfig] dEdX parameter updates: impact on PID (#55)
> 
>
>Following the recent updates on dEdX parameters from @MasakazuKurata in the Compute_dEdXProcessor config #53 , I wonder if the parameters provided in the PIDLikelihoodProcessor :
><parameter name="dEdxParameter_electron" type="FloatVec"> -2.40638e-03 7.10337e-01 2.87718e-01 -1.71591e+00 0.0
</parameter>
<parameter name="dEdxParameter_muon" type="FloatVec"> 8.11408e-02 9.92207e-01 7.58509e+05 -1.70167e-01 4.63670e-04
</parameter>
<parameter name="dEdxParameter_pion" type="FloatVec"> 8.10756e-02 -1.45051e+06 -3.09843e+04 2.84056e-01 3.38131e-04
</parameter>
<parameter name="dEdxParameter_kaon" type="FloatVec"> 7.96117e-02 4.13335e+03 1.13577e+06 1.80555e-01 -3.15083e-04
</parameter>
<parameter name="dEdxParameter_proton" type="FloatVec"> 7.78772e-02 4.49300e+04 9.13778e+04 1.50088e-01 -6.64184e-04
</parameter>  <parameter name="dEdxNormalization" type="float"> 1.350e-7 </parameter>  <parameter name="dEdxErrorFactor" type="float"> 7.55 </parameter>

**Comment by rete** [comment]
Code and config merged via pull request #53, #63 and [iLCSoft/MarlinReco#42](https://github.com/iLCSoft/MarlinReco/pull/42).
Next step is to reproduce result by using samples from (incoming) 3rd test MC production.

**Comment by rete** [comment]
Closing as superseeded by #75
